subjects/ascii-art/README.md

```diff
@@ -32,10 +32,10 @@ $$@@$$$$$$$$``````````@@$$@@$$$$$$
 ### Instructions
 
 - Your project must be written in **Go**.
-- The code must respect the [**good practices**](https://public.01-edu.org/subjects/good-practices/).
+- The code must respect the [**good practices**](https://((DOMAIN))/root/public/src/branch/master/subjects/good-practices/README.md).
```

Comment:
If you try to access the link through https://git.dev.01-edu.org/root/public/src/branch/project-reviews/subjects/ascii-art you will see that it doesnt work because this templating syntax only works in our frontend, not Gitea.
I suggest you try using relative links instead

Reply:
ok, so instead i could just use something like this (../good-practices/README.md) ?

Reply:
If it works, yes